views/applications.jade

```diff
@@ -3,15 +3,14 @@ extends layout
 block content
     .jumbotron.wicked-application-title
         .container.wicked-title-container
-            h1 Your Applications
+            h1 Register and View Your Applications
 
             p= glob.views.applications.titleTagline
 
     .container.wicked-container
 
         p.
-            An application is the entity to which you attach your API subscriptions. You can own or co-own
-            applications with other developers.
+            An application is the entity to which you attach your API subscriptions. You can own or co-own applications with other developers. Once you have registered an application, you can create and assign APIs that use it. You can assign only one application to an API.
```

Comment:
"You can assign only one application to an API." What does this mean? You can have multiple applications assigned to the same API, but one application can only be subscribed to an API once, is that what you mean?

Reply:
Indeed the text is confusing, will get back to you with revised text.